Makefile

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8; mode: makefile-gmake -*-
+
+.DEFAULT = help
+DEBUG ?= --pdb
+TEST ?= .
+
+# python version to use
+PY       ?=3
+# list of python packages (folders) or modules (files) of this build
+PYOBJECTS = hawkauthlib
+# folder where the python distribution takes place
+PYDIST   ?= dist
+# folder where the python intermediate build files take place
+PYBUILD  ?= build
+
+SYSTEMPYTHON = `which python$(PY) python | head -n 1`
+VIRTUALENV   = virtualenv --python=$(SYSTEMPYTHON)
+VTENV_OPTS   = "--no-site-packages"
```

Comment:
naming nit: can we call this a "VENV" rather than a "VTENV"?  Unless "VTENV" is a common naming convention elsewhere and I just haven't been keeping up with the times, which is entirely possible...

Reply:
right `VENV` is better / I will fix it.